rpmconf/Spec/Scripts/zimbra-ldap.post

```diff
@@ -24,14 +24,11 @@ mkdir -p /opt/zimbra/data/ldap/config
 mkdir -p /opt/zimbra/data/ldap/mdb/db
 chown -R zimbra:zimbra /opt/zimbra/data/ldap
 
-egrep -q '^%zimbra[[:space:]]' /etc/sudoers
-if [ $? = 0 ]; then
-	sudotmp=`mktemp -t zsudoers.XXXXXX 2> /dev/null` || { echo "Failed to create tmpfile"; exit 1; }
-	SUDOMODE=`perl -e 'my $mode=(stat("/etc/sudoers"))[2];printf("%04o\n",$mode & 07777);'`
-	egrep -v '^%zimbra[[:space:]]' /etc/sudoers > $sudotmp
-	mv -f $sudotmp /etc/sudoers
-	chmod $SUDOMODE /etc/sudoers
-fi
-chmod 440 /etc/sudoers.d/02_zimbra-ldap
-chown root:root /etc/sudoers.d/02_zimbra-ldap
 chown -R root:root /opt/zimbra/common/etc/openldap
+
+chown zimbra:zimbra /opt/zimbra/libexec/zmslapd
+chown zimbra:zimbra /opt/zimbra/common/libexec/slapd
+chown zimbra:zimbra /opt/zimbra/bin/ldap
```

Comment:
Why do you want the binaries to be owned by zimbra instead of root?
This again makes them vulnerable (writable) for attackers with local zimbra access.

Reply:
Fixed